### DISCUSSION TITLE
【生活】运动打卡 - 第4期 | ApacheCN

### DISCUSSION THREAD

**Discussion by lalatusiki** [apachecn/home#225] [Ideas]
> 活动介绍

- 日常生活中，如果你被如下事情困扰着：
  - **精力不足**，总是无精打采，**工作学习犯困**，每天都过得**身心疲惫**
  - 被**肥胖**困扰，颜值被耽误，甚至患上肥胖带来的一些列慢性病
  - **太瘦**没有美感，需要**增肌**
  - 想要养成运动健身的**好习惯坚持不下来**，总是中途放弃
  - **心理**正在承受着某些痛苦，**焦虑，抑郁，失恋，孤独**
- 加入活动，把自己**运动，控制饮食，休息放松，面对生活挑战**的情况记录分享出来给其他人制造压力吧

1. **本次活动【2022.11.09-2022.11.30】共三周，21天。**
2. 自己定制运动量，建议**循序渐进，感到身心愉悦**，不要一次运动就搞得筋疲力尽，浑身酸痛
3. **不要运动过量，运动不合理，伤害身体**（好好研究你的姿势对不对）
4. 一定要设计休息恢复日。休息日不是什么都不做。**轻量热身运动，瑜伽，泡沫轴筋膜枪按摩恢复都可以做**（大保健伤不伤身体？）
5. 饮食是核心，**增肌需要热量盈余，减脂需要热量缺口**。蛋白不停的损失，一定要多吃一些容易吸收的动物蛋白类食物。
6.  ** 请在活动群中寻找日记模板，记录在issue评论中 ** ，每日健身和习惯养成的情况请分享在群中

> 活动内容

- 坚持运动：定制符合自己的运动类型，运动量（避免照抄不适合自己的运动计划导致受伤）
  - 关注自己的心率是一个很好的办法，心率能有效反应你身体的状态，如果剧烈运动感觉坚持很难，那就让你运动时的心率保持在一百三以下吧。
  - hiit虽然减肥效果很好，但是心率长时间高速运转必然会让你身心疲劳想要放弃，请保持对心率的关注设定好休息和运动。
  - ** 如果你是因为很多年的习惯导致胖起来的，那么请不要追求很短时间就能变回去。**

- 控制饮食：吃什么，怎么吃真正决定了你的成果。
  - 不建议极端的生酮饮食（大部分人做不到撑到体质改变为生酮状态就会陷入崩溃开始暴饮暴食，没有碳水甚至会扰乱你的代谢）；
  - 不建议买那些代餐（没用还贵）。
  - 建议保持戒糖（改善肤质）
  - 规律饮食，低碳水和低油脂饮食（建议参考一个关于胰岛素的思路：把胰岛素当成一个身体储存脂肪的开关，开关打开你的身体就会进入低功耗储能模式，所以你要控制的是延长身体没有胰岛素分泌的时间；和降低你吃东西后太甜，让身体非常敏感的瞬间分泌很多胰岛素）；
  - 要健康饮食也要好吃（你能坚持健康饮食的原因不是因为能变瘦，一定是因为好吃）。
- 休息：合理的休息才会改变健身效果。还是那句注意关注自己的心率，胖人的心率会比瘦人高，所以也不要为自己心率较高而恐慌。
- 美型美容：大家都是人，你凭什么不可以护肤，美容？你凭什么可以不搭配好你的衣着？你凭什么可以不化妆？
- 坚持分享：监督其他人的完成情况，把你的运动数据分享给他们（向他们装逼）

> 活动qq群

573701357

**Comment by lalatusiki** [comment]
> 基本信息

昵称：兔斯基 

体重：81kg 

本期目标：减到77kg

> 第1天

- keep25分钟户外活力跑
- 早餐：豆渣饼，豆浆，两个水煮蛋。
- 午餐：白米饭+炒菜。
- 晚餐：苹果，火腿，一个火龙果

> 第2天

- 早晨：keep25分钟活力跑，27分钟，272千卡
- 早餐：鸡翅根4个，青椒鸡蛋炒饭，两个水煮蛋
- 午餐：凉皮一份，三根鸡翅根
- 晚餐：没有吃饭

> 第3天

- 体重：78.2kg
- 早晨：热身+keep25分钟活力跑。260千卡
- 早餐：鸡汤煮面条，两个水煮蛋
- 午餐：烧茄子，青尖椒胡萝卜艹蛋，白米饭两碗。
- 晚餐：芒果，樱桃，一袋牛奶

> 第4天

- 体重：78.6kg
- 早晨：热身+3公里法特莱克跑
- 早餐：
- 午餐：
- 晚餐：


> 第5天



> 第6天

- 5公里法特莱克跑步
- 25分钟户外活力跑

> 第7天

- 我能说什么，早晨我起不来。然后一天不想动

> 第8天

- 出门散步1个小时的渣渣，等于没运动。
- 这个沙雕县城，连个同龄女孩都见不到。害得我再ktv洗脚按摩一条街上多走了个来回，左顾右盼看向店铺里。
- 可能贫穷让这里的年轻姑娘白天上班，晚上ktv上班吧？？？

> 第9天

- 体重：77.5kg
- 跑步5.6公里，配速11
- 早餐：疙瘩汤，馒头
- 午餐，火腿蛋炒饭，一盒豆豉鱼罐头
- 晚餐：一杯蛋白粉（植物蛋白粉，一股维维豆奶的味道）

> 第10天

- 早晨：热身+keep25分钟活力跑。260千卡
- 夏天到了，肚子好大，比冬天还大。群友说如果不是胀气，那就是便秘。我每天早上都上卫生间，靠抖音通便。这情况莫不是？大便确实挺困难，我没必要狡辩了。或许我该给每次吃饭称重，每次排便称重（欸，好恶心阿）


> 第11天

- 一杯咖啡就能让你意识到原来你可以有这么多大便。肚子不鼓了
- 感冒变重了，我感觉我骚起来了

> 第12天

- 骚起来了，请假一天，躺床上睡觉。MD外面不卖退烧药。睡了一天就退骚了。

> 第13天

- 我恍惚间明白了中医说的精气神。因为近期运动所带来的气虚，我近四年来头一次被感冒搞到这个程度。这就是气虚，也就是所谓的抵抗力弱
- 补一补？

> 第14天

**Comment by novechina** [comment]
yeah ! just do it

**Comment by Mr-HHP** [comment]
> 基本信息

昵称：Mr.黄 

体重：67.5kg 

> 本期目标

锻炼核心力量和肩颈伸展，改善久坐敲代码带来的腰痛脖子僵硬

> 计划

周一至周五平板支撑腹部锻炼，周六不下雨晨练打球，周日休息日

> 第1天

周末休息日

> 第2天

早饭：半碗纯燕麦
午饭：青椒胡萝卜炒豆干+米饭
晚饭：一个土豆+半根胡萝卜+一小块鸡胸肉（煎的）+一个煮鸡蛋+一个红薯+两棵小青菜+一个西红柿

平板支撑7组（平均1分钟/组，间隔一分钟）
仰卧抬腿（15个/组）+仰卧收腹5组（20个/组）
结束动作：俯卧撑20个
腹部拉伸

> 第3天

早饭：半碗纯燕麦
午饭：茄子炒鸡腿肉+米饭
晚饭：一个土豆+半根胡萝卜+一小块鸡胸肉（煎的）+一个煮鸡蛋+一个红薯+两棵小青菜+一根黄瓜

平板支撑7组（平均1分钟/组，间隔1分钟）
仰卧抬腿（20/组）+仰卧收腿（20/组）
结束动作：俯卧撑20个
腹部拉伸

> 第4天

早饭：半碗纯燕麦
午饭：木耳炒鸡腿肉+米饭
晚饭：一个土豆+半根胡萝卜+一小块鸡胸肉（煎的）+一个煮鸡蛋+一个红薯+两棵小青菜+一个西红柿

平板支撑7组（平均1分钟/组，间隔1分钟）
仰卧抬腿（20/组）+仰卧收腿（20/组）
结束动作：俯卧撑20个
腹部拉伸

> 第5天

早饭：半碗纯燕麦
午饭：西红柿炒鸡蛋+米饭
晚饭：一个土豆+半根胡萝卜+一小块鸡胸肉（煎的）+一个煮鸡蛋+一个红薯+两棵小青菜+一根黄瓜

平板支撑7组（平均1分10秒/组，间隔1分钟）
仰卧抬腿（20/组）+仰卧收腿（20/组）
结束动作：俯卧撑20个
腹部拉伸

> 第6天

早饭：半碗纯燕麦
午饭：胡萝卜炒鸡腿肉+米饭
晚饭：土豆一个+胡萝卜半根+两块鸡胸肉+水煮蛋一个+红薯一根+青菜两棵+黄瓜一根

周五偷懒休息

> 第7天

晨练打球（9：30—11：00）：80分钟左右
下午出门步行：1万5千步左右

> 第8天

周末休息日

> 第9天

早饭：半碗纯燕麦
午饭：青椒炒千张+米饭
晚饭：土豆一个+红薯一个+青菜两棵+鸡胸肉一块+水煮蛋一个+胡萝卜半根+黄瓜一根

平板支撑6组（2分钟/组）
仰卧抬腿（20/组）+仰卧收腿（20/组）6组

> 第10天

早饭：半碗纯燕麦
午饭：火腿炒鸡蛋+米饭
晚饭：土豆一个+红薯一个+青菜两棵+鸡胸肉一块+胡萝卜半根+玉米一根

平板支撑7组（2分钟/组）
仰卧抬腿（20/组）+仰卧收腿（20/组）6组

> 第11天

偷懒休息

> 第12天

偷懒休息

> 第13天

部门团建聚餐，锻炼中断

> 第14天

早上9点-11.30，有氧运动，打篮球晨练

**Comment by Mr-HHP** [comment]
[Mr.黄计划.md](https://github.com/apachecn/home/files/6482939/Mr.md)

**Comment by jiangzhonglian** [comment]
> 基本信息

昵称：片刻

体重：55kg 

本期目标：增到65kg

> 第1天

周末休息

> 第2天

晚上：6组，80个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第3天

晚上：6组，100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第4天

晚上：6组，100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第5天

休息

> 第6天

晚上：6组，100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第7天

晚上：hiit 20分钟 + 3组，30个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第8天

晚上：100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第9天

休息

> 第10天

晚上：hiit 20分钟 + 3组，30个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第11天

晚上：6组，100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第12天

晚上：6组，100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第13天

休息

> 第14天

**Comment by limeicao** [comment]
> > 基本信息
> 
> 昵称：桑尘
> 
> 体重：48.5
> 
> 本期目标：到47kg
> 
>> 说些什么
> 
>减肥年年flag 
>> 6.6-6.10
> 
>立下flag,感觉又废了，日常嗜睡，说好的早上去游泳也不了了之，健身环也弃之不用，就是犯困。上个月还像打鸡血一样对运动乐此不疲。天天flag,天天倒。早餐基本两个鸡蛋一碗豆浆一碗牛奶，午餐及晚餐少吃碳水，其他正常。目前体重稳如泰山。
>> 第六天 6.11
> 
>早餐  两个蛋清 +牛奶+豆浆
午饭 减肥餐
运动  健身房跑步5km(待完成)

**Comment by jiangzhonglian** [comment]
> 基本信息

昵称：片刻

体重：55kg 

本期目标：增到65kg

> 第1天

周末休息

> 第2天

晚上：hiit 15分钟 + 3组，30个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第3天

晚上：hiit 15分钟 + 3组，30个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第4天

休息

> 第5天

晚上：100个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第6天

晚上：hiit 20分钟 + 3组，40个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第7天

休息

> 第8天

休息

> 第9天

晚上：hiit 20分钟 + 3组，30个俯卧撑
晚餐：2个香蕉，1个酸奶

> 第10天

晚上：hiit 15分钟 + 3组，30个俯卧撑
晚餐：2个香蕉

> 第11天

休息

> 第12天

休息

> 第13天

晚上：100个俯卧撑
晚餐：2个香蕉

> 第14天

晚上：hiit 20分钟
晚餐：2个香蕉，1个酸奶

**Comment by lalatusiki** [comment]
健身打卡-第二期



> 基本信息

昵称：兔斯基 

体重：78.8kg 

本期目标：减到77kg

> 本期其实快达到目标了，中间有四天左右的懈怠和停顿，好在有着群友的督促和讨论。我又开始继续坚持。目前实践给我的经验是：
>
> 1.我必须晚上十点睡觉，才能维持我五点五点半起床。
>
> 2.我必须每天早上运动一会儿，甚至不一定是跑步，让我感到一种豁然开朗的感觉（就像第六天打卡时说的一样：感觉自己就像突然打开了什么开关，身体突然就开始活力涌现）
>
> 3.要想肚子不胀气，我就得吃比原来少很少的食物，并不会造成我饿肚子，但是很显然胀气是因为我的消化能力早就没有吃那么多就能立马消化完的强大了



> 第1天 2021-6-6

- 肚子胀气，无法解决
- 囚徒健身-斜上俯卧撑 x 15 x 三组
- 囚徒健身-深蹲 x 30 x 三组
- 说好了戒撸，其实我还是偷偷看小h图

> 第2天 2021-6-7

- 过去我是由碗的量来限制饮食的
- 现在发现在没有计量秤的情况下，按目视直观的体积更容易
- 米饭一拳头大小，菜一拳半大小
- 不知怎么的，为了戒色，删掉了所有的网址和图片视频，还把库存硬盘锁在了柜子里。但是我还是默默的打开，接上线，插在了电脑上

> 第3天 2021-6-8

- 早晨去跑步，发现在路上跑比在公园里好的地方。本来看到别人我都会感到紧张和尴尬。在路上跑人就显得距离比较远了，反而这种紧张尴尬感就小了
- 饮食真的很难控制好，干了三碗面条.....。吃这个东西，容易上瘾，如果我不对这个上瘾，那一定是我在其他事情上感到快乐
- 不知道哪个家伙说的想让我发网址给他，我就把硬盘拿出来了，戒色继续失败

> 第4天 2021-6-9

- 你们知道四君子汤嘛？听说长得胖的人要补气。可真TM难喝阿，难怪滞销呢。
- 我用郫县豆瓣酱+火腿肠+蒜苔青椒炒了酱汁，拌面吃。就吃多了，以后绝对不要自己做酱汁
- 我曾经以为，今天是戒撸第一天。谁知道它竟然不是呢。或许是明天呢。

> 第5天 2021-6-10

- 从凌晨开始拉肚子，这一夜我成功的洗肠子了。困得不行，运动个屁。办公室里也是坐在沙发上疲惫的一天。
- 饭量控制的不错，就是晚上拿卫龙辣条当晚餐是不是不太对？
- 一直在steam打火影，这个游戏在线联机我就没赢过.....
- 开始做冥想练习，为了清理掉自己杂乱的什么都想做的念头。我开始注意到，自己对某个女生的仇恨度原本就是她在我思想中的干扰程度造成的。果然女人只会影响到我拔刀的速度。

> 第6天 2021-6-11

- 早起去跑步，三十分钟左右我终于感受到了运动带来的快感，这种快乐是其他娱乐（打游戏、打飞机、群里闲聊、胡吃海塞）所没有的。就像是我跑了三十分钟，一开始总是感觉腿酸啦，不爽啦，懒惰不想动啊。三十分钟一过，身体就像打开了某种激素的释放开关，我感到心情舒畅。总想不由自主的跑起来。5公里，gps信号不好显示我飞过去的直线距离。
- 下午又按着这个路线出去跑步，大概是当天跑累了，难以再找到那个呼吸心跳和运动幅度的平衡点，快乐并没有随之而来。此次总共7.36公里。


> 第7天 2021-6-12

- 没怎么运动，研究中药改善身体的办法。脑子里始终挥之不去的就是那些，中医讲的五脏健康身体气血正常是不可能胖的，身体自己就恢复正常了。
- 六味地黄丸味道一般般，人参归脾丸就很沙雕了，一股白萝卜味儿。什么逍遥丸，香砂养胃丸都是清一色的酸味。
- 每逢周末，躺在床上打瞌睡，要么起床打游戏。打游戏也解决不了困乏的时候，脑子就想让你撸，毕竟他只是想找个能兴奋起来的办法而已，脑子有什么错呢。
- 一整天的休息带来的是更累了，所以，**不该睡觉的时候不要躺在床上，不然一天都是浑噩疲惫的。**

> 第8天 2021-6-13

- 早晨简单活动活动，对于囚徒健身越来越感到不屑，狗屁玩意儿得练到什么时候。但也是觉得，这种低心率的无氧运动真稀奇。或许真的是高燃脂又增肌了吧。
- 下午出门练练开车，一如既往的路上没啥事，停车的时候把大爷吓坏了，连忙开着自己的五菱小电车离开了停车场。
- 值得反思的问题：如何才能让心态平稳健康的应对生活中出乎计划和心理准备事情发生？如何在不得不同时处理多件事时，能够不让自己陷入焦急愤怒（感觉是天生的，当事情繁杂需要意志力的时候，我的眉头会逐渐的皱起来）
- 我想说今天戒撸第一天，不过现在看来又是明天了

> 第9天 2021-6-14

- 当你前一个晚上早早的老实睡觉了，今天能完成日常计划的可能性就大大提高了
- 今天成功戒撸，并且出去跑了一个小时的步，感受着满身的汗水，真爽

> 第10天 2021-6-15

- 上班第一天
- 戒撸第二天
- 又一次跑了个步，前期总会有一会儿难以进入跑步的状态，并且会感到小腿前侧和膝盖的酸痛。当跑了一会儿状态到了之后就不再痛了。我想这大概就是一种热身不够充分的表现

> 第11天 2021-6-16

- 少跟人聊不该聊的，我的戒色大计又重新开始了，啥沙雕网友，欠抽，啪~
- 撸过之后，我会觉得自己需要营养，而不是消耗营养。狂吃成了问题。

> 第12天 2021-6-17

- 苦恼了我有把个月的肚胀问题，终于研究出了方法
  - 晚上不吃就行了，然后你就会发现自己晚上一直在放屁，被子像旗杆上飘扬的🚩
- 其实，有大佬帮我研究思考用别的中药，主要是我买的四君子汤都是垃圾，没药效的药
- **上网搜中药养生的药时也发现了分辨办法，很多药都其实能立竿见影的，凡是评论里吃了几顿没效果，不管他们有没有找这个需要长时间服用的理由，都是骗人的。**

> 第13天 2021-6-18

- 戒色忍不住很正常，但是我跟群友耗上了，我非得超过他们，双手打字以证清白。
- 不是跑步，改用keep的突击减脂课程，每天都是简单的不喘气的动作。大大减少了我对手机上健身操的方案程度。同时加上合理的饮食控制，我的体重终于下了78kg，77.6kg 纪念一下，没有贪吃的想法，也没有暴食的三餐。或许真的是因为成年人早就不能像青春期孩子一样的狂吃了，消化系统早就没有那么强大了。

> 第14天 2021-6-19

- 约了女孩子吃晚餐，本来我已经不再吃晚餐了，烤肉带来的后果绝对是刚下来的体重弹回去
- 健身的好处就是改变你的精神状态，也改变你的外形，女生说我本人比照片好看....我当时就有点不乐意，不爱拍照的我花了半个多小时才用型男相机拍出来的照片啊。哈哈哈哈哈
- 餐馆里的食物肯定是有问题的，我在床上躺了比以往要久的多的时间才能入睡

**Comment by sandexp** [comment]
###### 基本信息

昵称：鸡⑧

目标：减减肥, 提高颜值, 找到合适自己的地方

> 计划：
>
> 1. 每天晚上7:30-8:00 有氧30分钟
> 2. 早上7:00起床, 散步30分钟
> 3. 饮食上拒绝高热量食物, 可以使用水冲饥. 保证三餐正常摄入能量.
> 4. 在3的基础上, 每三天省略一天晚上的进食.
>
> 目标:
>
> 14天减重3kg体重

> ###### 第1天 2021-6-21

​		进度:

1. 运动量正常

2. 无传统手艺活动

3. 7点起床, 12点睡眠, 睡眠正常

4. 未注意着装

5. 吃了2包方便面, 属于高热量食物

6. 未调研正确的饮食计划

   达成指标数量: 3

   未达成指标数量: 3

> ###### 第2天 2021-6-22

​		进度:

1. 运动量正常

2. 无传统手艺活动

3. 8点起床, 12点睡眠, 睡眠正常

4. 未注意着装

5. 吃了1包方便面, 属于高热量食物


   达成指标数量: 3

   未达成指标数量: 2

**Comment by lalatusiki** [comment]
## 健身打卡-第三期



> ###### 基本信息

昵称：兔斯基 

目标：证明自己是一个喜欢运动的人，除了买了那一堆运动装备，我每天都在运动。养成每天都或多或少运动的习惯

> 计划：
>
> ​	1.每天五点半起床后，我都会做一定量运动唤醒全身，到那种似乎打开了什么开关给精神充了电的感觉。
>
> ​	2.每天下班后，无事可做的情况下，运动到微微出汗，有一种快乐的感觉，替代平时一无聊就打游戏的情况。

> ###### 本期结束总结：
>
> 1.运动不能解决所有问题，但是它带来的就是一种快乐。
>
> 2.咖啡牛逼，一直喝一直爽。喝太多了火葬场。
>
> 3.群里和GitHub  issue的热度完全不是一个量级。我感觉活动办到第三期，本身缺了点什么，缺少一种feel，一种情趣感。一宗情绪，对就是少了点东西，是精神上的缺少了啥。我该怎么修改活动来让所有人找到一种特殊的情绪呢？



> ###### 第1天 2021-6-21

- 

> ###### 第2天 2021-6-22

- 户外跑步4.83公里

> ###### 第3天 2021-6-23

- 

> ###### 第4天 2021-6-24

- 边看电影边跳各种动作，搏击操或者其他什么混着来
- 电影还是得选有帅哥的露肉比较多的搏击动作片，电影的渲染力令人佩服

> ###### 第5天 2021-6-25

- 继续播放突袭，边看人家打架，自己再一旁蹦
- 突然想起了一些人好像不健身，光看人家的健身视频然后就给自己打了不少鸡血，视频看完了，其实也没运动。但是心理满足感却爆棚？？？

> ###### 第6天 2021-6-26

- 网卡了，电影播不完，运动也就没动力？？？
- 我一定是忘了想象自己瘦下来有多棒，所以才没有动力。


> ###### 第7天 2021-6-27

- 

> ###### 第8天 2021-6-28

- 以为要下雨，室内开合跳600
- 最后看天气出去跑步4公里，我感到了我好像弄伤了自己

> ###### 第9天 2021-6-29

- 

> ###### 第10天 2021-6-30

- 晚餐我不吃了，欸就是不吃，就是饿着

> ###### 第11天 2021-7-1

- 马镫工装的包真渣，连个拉链都没。虽然说是真TM好看。但是我把春夏秋冬的衣服从正装点的polo衫和衬衣，到休闲的风衣羽绒服全试了个遍。汗水沾满了我全身。我感觉这算是运动吧。毕竟我心很累啊，考虑退不退货的那种累

> ###### 第12天 2021-7-2

- 

> ###### 第13天 2021-7-3

- 我喝了一瓶体制能量。一瓶黑糖咖啡。然后一整个下午陷入不明觉厉的屎感。最后我都只剩水了。
- 过去以为肚子大是脂肪多，这次连续上厕所，肚子小了。我明白了是因为水。体重也从上午79掉到下午的77。
- 咖啡牛逼啊，我不得不感叹。

> ###### 第14天 2021-7-4

- 打了一整天的彩虹六号，两杯黑咖啡喝的我心慌，知乎说是咖啡碱太多导致的

**Comment by KangChenDanika** [comment]
> 基本信息 

 昵称：53 小康
 目标：瘦到100斤，或者瘦成100斤的样子

> 计划：

早上6：30—7:30  跳绳+跳操
晚上9：00—10:00  跑步+力量训练
（量尽力就好，多少看心情和极限）

> 第一天 2022-11-9

- 早上：跳绳1000个
- 早餐：一笼包子
- 晚上：骑行10公里
- 晚餐：大盘鸡拌面

> 第二天 2022-11-10

- 早上：跳操25min、跳绳1100个
- 早餐：紫薯包+豆浆
- 晚餐：紫薯包+燕麦粥
- 晚上：跑步5公里

> 第三天 2022-11-11

- 早上：起晚了
- 早餐：玉米（没有一点玉米味...）+豆浆
- 晚餐：自选餐（高热量的，好好吃，不过要少吃）
- 晚上：步行2公里

> 第四天 2022-11-12

- 早上：睡懒觉
- 早餐：馅饼+豆浆（馅饼有点油）
- 晚餐：自选餐（又吃了高热量的食物，不过吃的很少）
- 晚上：跑步400米
- 其他：（1）拿到1000块钱奖学金 （2）发现很多时候缺乏面对自己缺点和问题的勇气

> 第五天 2022-11-13

- 早上：睡懒觉
- 早餐：无
- 晚餐：大米（只吃了两口大米，以后吃大米只吃菜！一定要瘦！）
- 晚上：跑步3公里

> 第六天 2022-11-14

- 早上：跳操10min、跳绳1400个
- 早餐：5口面包
- 晚餐：米线
- 晚上：8点就睡了

> 第七天 2022-11-15

- 早上：跳操10min+跳绳2000个
- 早餐：一个半麻球（超级好吃）+小米粥
- 晚餐：自助餐（没有吃大米）
- 晚上：跑步10公里

>第八天 2022-11-16

- 早上：跳操3min
- 早餐：牛肉卷饼（好吃到爆）+1.5块面包（吃的有点多了）
- 晚餐：一份大米+一杯粥+一杯奶茶+三个泡芙+一块饼干（暴饮暴食）
- 晚上：休息

> 第九天 2022-11-17

- 早上：休息
- 早餐：一个面包+一个蛋黄酥
- 晚餐：8块钱自助+羊肉卷饼（暴饮暴食）
- 晚上：休息

> 第十天 2022-11-18

- 早餐：一根火腿
- 早上：休息
- 晚餐：一份大米（暴饮暴食，明天不能这样了，一定要瘦！）
- 晚上：休息

> 第十一天 2022-11-19

- 基本上啥也没干

> 第十二天 2022-11-20

- 早上：休息
- 早餐：两个紫薯包+燕麦粥（应该吃一个的，第二个的时候美味就流失了...）
- 晚餐：
- 晚上：
- 其他：我的朋友怎么能这么优秀，不知道啥时候追上人家，怎么做到一直努力的？

> 第十三天 2022-11-21

- 学校疫情爆发，不敢出去

> 第十四天 2022-11-22

- 早餐：一个面包
- 早上：无

> 第十五天 2022-11-26

- 早餐：稀饭
- 早上：跳操 40min
- 晚上：跳操 40min
- 晚餐：一个包子+一碗粥
- 其他：gap了几天，因为放假了回家一点不想动，要积极改变

> 第十六天 2022-11-27

- 早餐：3个土豆饼（纯油炸，我有点狂）
- 早上：跳操30min
- 晚餐：白萝卜汤（老妈说巨减肥，未来几天都喝白萝卜汤）

> 第十七天 2022-11-28

- 早上：跳操15min+跳绳600个
- 早餐：鸡蛋汤
- 晚上：
- 晚餐：白萝卜汤
- 其他：白萝卜汤好厉害，体重下降4斤，为什么？

**Comment by jiangzhonglian** [comment]
1
